recipes/r-n1qn1/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -o errexit -o pipefail
+if [[ ${target_platform} =~ linux.* ]] || [[ ${target_platform} == win-32 ]] || [[ ${target_platform} == win-64 ]] || [[ ${target_platform} == osx-64 ]]; then
+  export DISABLE_AUTOBREW=1
+  ${R} CMD INSTALL --build .
+else
+  mkdir -p "${PREFIX}"/lib/R/library/n1qn1
+  mv ./* "${PREFIX}"/lib/R/library/n1qn1
+  if [[ ${target_platform} == osx-64 ]]; then
+    pushd "${PREFIX}"
+      for libdir in lib/R/lib lib/R/modules lib/R/library lib/R/bin/exec sysroot/usr/lib; do
+        pushd "${libdir}" || exit 1
+          while IFS= read -r -d '' SHARED_LIB
+          do
+            echo "fixing SHARED_LIB ${SHARED_LIB}"
+            install_name_tool -change /Library/Frameworks/R.framework/Versions/3.5.0-MRO/Resources/lib/libR.dylib "${PREFIX}"/lib/R/lib/libR.dylib "${SHARED_LIB}" || true
```

Comment:
If these patches do anything, the package is not being built correctly. These would mean that the wrong libraries are being found at build time. It would be good to remove all of these and then make sure none of these libraries are linked.

Reply:
Thanks for your input! Testing this right now.

Reply:
Changes nothing, reverting. We'll try with fresh rebuild of R3.5.1

Reply:
These lines are added by the cran skeleton, so I am not sure that is an issue.